src/patch_QEDbase.jl

```diff
@@ -4,6 +4,3 @@
 # See https://github.com/QEDjl-project/QEDbase.jl/issues/69
 struct UnknownDirection <: QEDbase.ParticleDirection end
 Broadcast.broadcastable(d::UnknownDirection) = Ref(d)
-
-is_incoming(::UnknownDirection) = false
```

Comment:
Why removing just this? Is this patched upstream? Why not removing the whole thing?

Reply:
It's an open issue, so not in QEDbase yet as far as I'm aware